eclipse-scout-chart/src/chart/ChartJsRenderer.js

```diff
@@ -1101,7 +1101,12 @@ export default class ChartJsRenderer extends AbstractChartRenderer {
       value = dataset.data[dataIndex];
 
     if (this._isHorizontalBar(config)) {
-      tooltipDirection = value < 0 ? 'left' : 'right';
+      if (!numbers.isNumber(value) && objects.isArray(value.x) && value.x.length === 2) {
```

Comment:
```suggestion
      if (value && objects.isArray(value.x) && value.x.length === 2) {
```
If the value is an array it can not be a number.